src/components/OrgListCard/OrgListCard.tsx

```
@@ -57,15 +57,15 @@ function orgListCard(props: InterfaceOrgListCardProps): JSX.Element {
       organizations: InterfaceQueryOrganizationsListObject[];
     };
   } = useQuery(ORGANIZATIONS_LIST, {
-    variables: { id: _id },
+    variables: { id: id },
   });
 
   // Handle click event to navigate to the organization dashboard
   function handleClick(): void {
-    const url = '/orgdash/' + _id;
+    // const url = '/orgdash/' + id;
 
```

ESLint

[error] Delete `⏎`

(prettier/prettier)